examples/cuda/examples/bulk.cpp

```diff
@@ -0,0 +1,36 @@
+#include <schedulers/stream.cuh>
+#include <execution.hpp>
+
+#include <cstdio>
+
+namespace ex = std::execution;
+namespace stream = example::cuda::stream;
+
+int main() {
+  using example::cuda::is_on_gpu;
+
+  stream::context_t stream_context{};
+  ex::scheduler auto sch = stream_context.get_scheduler();
+
+  auto bf = [](int lbl) {
```

Comment:
Given that these are example functions I would appreciate some more descriptive names.

`bulk_function` and `then_function` are not great but better than `bf` and `tf`